src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.io.File;
+
+public class Main {
+    public static void main(String[] args) {
+        WorkWithFile workWithFile = new WorkWithFile();
+
+        String outputDirectory = "output";
+
+        File directory = new File(outputDirectory);
+        if (!directory.exists()) {
+            directory.mkdirs();
+        }
+
+        String[] inputFiles = {"apple.csv", "banana.csv", "grape.csv", "orange.csv"};
+        for (String inputFile : inputFiles) {
+            String outputFile = outputDirectory + "/" + inputFile.replace(".csv", "_output.csv");
+            workWithFile.getStatistic(inputFile, outputFile);
+        }
+    }
+}

```

Comment:
it seems to be a code you've used for testing purposes. It shouldn't be included to the PR of this task. As well as your entire output folder